modules/auxiliary/scanner/lotus/lotus_domino_hashes.rb

```diff
@@ -157,9 +157,10 @@ def dump_hashes(view_id, cookie, uri)
       }, 25)
 
       if res && res.body
-        short_name = res.body.scan(/<INPUT NAME=\"ShortName\" TYPE=(?:.*) VALUE=\"([^\s]+)"/i).join
-        user_mail = res.body.scan(/<INPUT NAME=\"InternetAddress\" TYPE=(?:.*) VALUE=\"([^\s]+)"/i).join
-        pass_hash = res.body.scan(/<INPUT NAME=\"\$?dspHTTPPassword\" TYPE=(?:.*) VALUE=\"([^\s]+)"/i).join
+        doc = res.get_html_document
+        short_name =  doc.xpath('//input[@name="ShortName"]/@value').text
+        user_mail =  doc.xpath('//input[@name="InternetAddress"]/@value').text
+        pass_hash = doc.xpath('//input[@name="$dspHTTPPassword" or @name="dspHTTPPassword"]/@value').first&.text
```

Comment:
Will await the http logs, but we should just double check the case sensitivity around the node name of `input` or `INPUT`